app/code/Magento/Backend/view/adminhtml/templates/dashboard/totalbar.phtml

```diff
@@ -3,19 +3,28 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+
+use Magento\Backend\Block\Dashboard\Totals;
+use Magento\Framework\Escaper;
+
+/**
+ * @var Totals $block
+ * @var Escaper $escaper
```

Comment:
Hi @Bartlomiejsz,
I'm not sure I've ever saw such example. Could you add more details how it works, or where can I read more details?

Reply:
Hi @ihor-sviziev, escaper methods in AbstractBlock were deprecated some time ago, please check this commit: https://github.com/magento/magento2/commit/b5fde42dacbca4239e9276f871fc6b48840a28a7#diff-0e10c44d6b0961dcab05891b110d0305

Reply:
Thank you for pointing me! Looks really good!